tooling/ast_fuzzer/src/program/mod.rs

```diff
@@ -546,7 +546,7 @@ fn make_name(mut id: usize, is_global: bool) -> String {
     }
     name.reverse();
     let mut name = name.into_iter().collect::<String>();
-    if matches!(name.as_str(), "as" | "if" | "fn" | "for" | "loop") {
+    if matches!(name.as_str(), "as" | "if" | "in" | "fn" | "for" | "loop") {
```

Comment:
Is this meant to be a complete keyword list? We could copy many more from tokens.rs